src/layer.js

```diff
@@ -220,6 +220,41 @@ export default class LayerClient {
     }
   }
 
+  /**
+   * Get detailed information about a featureType.
```

Comment:
Pleas use `FeatureType` notation (with upper case). Also for the API doc notations below.

Reply:
done with [8a70879](https://github.com/meggsimum/geoserver-node-client/pull/71/commits/8a7087989a8d9f88f8017a6554f9730a0a4faef1)